### PR TITLE
Add cli, toolset, and core-sdk mirrors to dotnet vsts instance

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -646,8 +646,15 @@
         "https://github.com/dotnet/arcade/blob/release/**/*",
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/diagnostics/blob/release/**/*",
+        "https://github.com/dotnet/core-sdk/blob/master/**/*",
+        "https://github.com/dotnet/core-sdk/blob/release/**/*",
+        "https://github.com/dotnet/cli/blob/master/**/*",
```
Comment (livarcocc) [inline Maestro/subscriptions.json]: I was under the impression we had CLI already.

Comment (mmitche) [reply Maestro/subscriptions.json]: Only in the devdiv account.  This is for the dotnet account.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -646,8 +646,15 @@
         "https://github.com/dotnet/arcade/blob/release/**/*",
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/diagnostics/blob/release/**/*",
+        "https://github.com/dotnet/core-sdk/blob/master/**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: I'm going to go ahead and merge this, might be nice to keep these trigger lists alphabetized in the future though.

**Pull request merged**